app/utils/auth.ts

```diff
@@ -53,32 +54,33 @@ export const formatRoleName = (role: string): string => {
 };
 
 /**
- * Hook which retrieves the user's bearer token from the logged-in
- * user's IdToken claims.
- *
- * Todo: our Auth0 app is configured to return idTokens, which are
- * "opaque" proprietary Auth0 tokens, seemingly because this enables us
- * to interact with the User Management API. as a result, we must
- * use idToken.__raw to grab the actual JWT. it seems like our setup
- * may be misconfigured, because accessing .__raw seems like a hack :/
```

Comment:
This PR is accompanied by a change to our Auth0 post-login action, which I describe [here](https://github.com/cityofaustin/atd-data-tech/issues/20391#issuecomment-2558161548). Fixing the Auth0 configuration enables us to use the normal `getAccessTokenSilently()` and we can do away with `getIdTokenClaims()`

Reply:
where did you update post-login action, is that configured in auth0 itself?

Reply:
Yes. [Here is a direct link](https://manage.auth0.com/dashboard/us/atd-datatech/actions/library/details/3ea62805-ea37-465e-b0a5-1b1ef2bd26c8) to our **Set Hasura Claims**  post-login action. I also added this link to the my last comment on the issue.